admin-site/src/OccasionsList.svelte

```diff
@@ -54,30 +45,18 @@
 
   function occasionButton(id) {
```

Comment:
I don't have a problem with this, but it would be sliiiightly more common / customary to keep the markup super simple (i.e. just let on:click call its handler and pass the usual `DOMEvent` object automatically), and then in your handler function, grab `event.target.id`. (https://developer.mozilla.org/en-US/docs/Web/API/Event/target)

Reply:
I had to use event.currentTarget.value. I believe that's because we have the Button component calling the event.